client/src/components/Popover/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import useStyles from './style';
+
+const Popover = () => {
+  const classes = useStyles();
+  return (
+    <>
+      <Box className={classes.btns_container}>
+        <Link to="/food" className={classes.links}>
+          <Button variant="outlined" className={classes.popover_btns}>
+            🥗 food
```

Comment:
If you have ESLint errors because of the emojis in this file, this is ESLint's preferred fix:
`<span role="img" aria-label="food">🥗</span>`

This is better for accessibility (users with disabilities). You can read more here (if you're bored): https://medium.com/@seanmcp/%EF%B8%8F-how-to-use-emojis-in-react-d23bbf608bf7